playbooks/preflight-check.yml

```diff
@@ -175,3 +175,50 @@
     - name: Duplicate flavors exist
       fail: msg="Duplicated flavor IDs are detected."
       when: flavor_check.stdout != ""
+
+# RLM-1372 Check NFS exsiting
```

Comment:
do we ALWAYS want to fail if glance has NFS, seems like this is a valid use-case.